pkgs/development/python-modules/aioimaplib/default.nix

```diff
@@ -15,6 +17,9 @@
 buildPythonPackage rec {
   pname = "aioimaplib";
   version = "0.9.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.5" || pythonAtLeast "3.10";
```

Comment:
```suggestion
  # https://github.com/bamthomas/aioimaplib/issues/75
  disabled = pythonOlder "3.5" || pythonAtLeast "3.10";
```

Reply:
The `setup.py` file already indicates it supports from 3.5 to 3.9. Is it necessary to also point this issue?

Reply:
I find it useful because it's a straightforward reference for checking whether Python 3.10 support was added.

Reply:
Makes sense. I will add a description to indicate the reason of the link, for future update references.